lib/puppet/util/run_mode.rb

```diff
@@ -38,6 +38,10 @@ def log_dir
         "$vardir/log"
       end
 
+      def conf_dir
+        "$puppetdir/config"
+      end
+
```

Comment:
On Wed, Feb 04, 2015 at 11:06:47AM -0800, Chris Price wrote:
> > @@ -38,6 +38,10 @@ def log_dir
> >          "$vardir/log"
> >        end
> >  
> > +      def conf_dir
> > +        "$puppetdir/config"
> > +      end
> > +
> 
> Yeah, I think I agree with that statement, but it seems like it would imply that the log_dir value a few lines earlier should be equally broken.
> 

I don't think so

we NEVER use Puppet::Util::RunMode[:master].log_dir directly. However it
is Puppet::Util::RunMode[:master].conf_dir that is bypassing the
extrapolate stuff.